packages/core/src/compiler/Style.ts

```diff
@@ -2070,7 +2070,7 @@ const findPathsExpr = <T>(expr: Expr<T>): Path<T>[] => {
       return expr.properties.flatMap((prop) => findPathsExpr(prop.value));
     }
     case "Layering": {
-      return [expr.below, expr.above];
+      return [expr.left, ...expr.right];
```

Comment:
The reason `diagrams/` are being updated is because of this line. If you change it to the following, behavior is the same as before:
```suggestion
      if (expr.layeringOp === "below") return [expr.left, ...expr.right];
      else return [...expr.right, expr.left];
```
But I don't think we should actually add this complexity just to maintain the exact same behavior. Could you update the `variation` for `diagrams/siggraph-teaser-euclidean-teaser.svg` to make it look nice again, though?

Reply:
Agreed. Perhaps it's a bit odd that a users changes from `above` to `below` and sees diagram changes in the IDE. How significant is the ordering here? Can we normalize it somehow?

Reply:
I think this should be part of a larger discussion about what kinds of stability we should try to support beyond raw determinism.

Reply:
Sounds good. Let's talk more about this later. Merging this PR now.